apps/client/src/components/Nav.tsx

```diff
@@ -1,13 +1,23 @@
 import { ChevronLeft } from "@datapunt/asc-assets";
 import { Button } from "@datapunt/asc-ui";
-import PropTypes from "prop-types";
 import React from "react";
 
 import { PrevButton } from "../atoms";
 import { NEXT_BUTTON } from "../utils/test-ids";
 import { IconContainer, IconLeft, NavStyle } from "./NavStyle";
 
-const Nav = ({
+export type NavProps = {
+  formEnds: boolean;
+  nextText: string;
+  noMarginBottom: boolean;
+  onGoToNext: (event: React.FormEvent) => void;
+  onGoToPrev: (event: React.FormEvent) => void;
+  prevText: string;
+  showNext: boolean;
+  showPrev: boolean;
```

Comment:
I think all these props are optional and now they're required. If you make them optional, you can remove the default values from `formEnds` and `noMarginBottom`.

And since there are only two default props left, you can remove the `Nav.defaultProps` and place those two in the top.